src/Modules/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/editLanguagePanel/resourceEditor/index.jsx

```diff
@@ -87,9 +87,7 @@ class ResourceEditor extends Component {
             inFullMode: false
         });
 
-        setTimeout(() => {
-            window.dnn.stopEscapeFromClosingPB = false;
-        }, 500);
+        window.dnn.stopEscapeFromClosingPB = false;
```

Comment:
Every other `setTimeout` was wrapping access to a `this`. Here is the only case where the callback accesses the global scope. Isn't this one actually safe?